test_settings.py

```diff
@@ -102,6 +107,8 @@ def noop_gettext(s):
         'django.middleware.csrf.CsrfViewMiddleware',
         'django.middleware.locale.LocaleMiddleware',
         'django.middleware.common.CommonMiddleware',
+        # NOTE: This will actually be removed below in CMS<3.2 installs.
+        'cms.middleware.utils.ApphookReloadMiddleware',
```

Comment:
To keep consistency, I recommend putting this at the top as done by https://github.com/aldryn/aldryn-django-cms/blob/support/3.2.x/aldryn_config.py#L92